core/frontend/src/views/ExtensionView.vue

```diff
@@ -27,10 +34,13 @@ export default Vue.extend({
     fullpage(): boolean {
       return this.$route.query.full_page === 'true'
     },
-    port(): number {
+    port(): number | undefined | null {
```

Comment:
is it obvious what it means? or should I add a comment?
number = port
undefined = we dont know (still loading)
null = nope